modules/inmarBidAdapter.js

```diff
@@ -0,0 +1,113 @@
+import * as utils from '../src/utils.js';
+import { config } from '../src/config.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'inmar';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['inm'],
+  supportedMediaTypes: [BANNER, VIDEO],
+
+  /**
+   * Determines whether or not the given bid request is valid
+   *
+   * @param {bidRequest} bid The bid params to validate.
+   * @returns {boolean} True if this is a valid bid, and false otherwise
+   */
+  isBidRequestValid: function(bid) {
+    return !!(bid.params && bid.params.partnerId && bid.params.adnetId);
+  },
+
+  /**
+   * Build a server request from the list of valid BidRequests
+   * @param {validBidRequests} is an array of the valid bids
+   * @param {bidderRequest} bidder request object
+   * @returns {ServerRequest} Info describing the request to the server
+   */
+  buildRequests: function(validBidRequests, bidderRequest) {
+    var payload = {
+      bidderCode: bidderRequest.bidderCode,
+      auctionId: bidderRequest.auctionId,
+      bidderRequestId: bidderRequest.bidderRequestId,
+      bidRequests: validBidRequests,
+      auctionStart: bidderRequest.auctionStart,
+      timeout: bidderRequest.timeout,
+      refererInfo: bidderRequest.refererInfo,
+      start: bidderRequest.start,
+      gdprConsent: bidderRequest.gdprConsent,
+      uspConsent: bidderRequest.uspConsent,
+      currencyCode: config.getConfig('currency.adServerCurrency'),
+      coppa: config.getConfig('coppa'),
+      firstPartyData: config.getConfig('fpd'),
+      prebidVersion: '$prebid.version$'
+    };
+
+    var payloadString = JSON.stringify(payload);
+
+    return {
+      method: 'POST',
+      url: 'https://prebid.owneriq.net:8443/bidder/pb/bid',
+      options: {
+        withCredentials: false
```

Comment:
Confused as to why you are sending withCredentials as false since that makes user sync pointless since you wont get your cookie back on the bid request